switchmap/poller/async_poll.py

```
@@ -0,0 +1,277 @@
+                    pprint(data)
+
+                return True
+            else:
+                log_message = (
+                    f"Device {hostname} returns no data. Check "
+                    f"connectivity/SNMP configuration"
+                )
+                log.log2debug(1408, log_message)
+                return False
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)